example/package.json

```diff
@@ -7,18 +7,19 @@
   "description": "webpack-config example app",
   "main": "webpack.config.js",
   "dependencies": {
+    "extract-text-webpack-plugin": "^2.0.0-rc.3",
+    "html-webpack-plugin": "^2.28.0",
```

Comment:
HTML Webpack plugin should be part of the root package dependency. As a user of `ca-webpack-config`, I don't want to have to specify webpack plugins that are in `ca-webpack-config`.